build.gradle

```diff
@@ -1,29 +1,24 @@
 plugins {
-    id "java-library"
-    id "scala"
-    id "maven-publish"
-    id 'fabric-loom' version '0.10.30'
-    id "com.matthewprenger.cursegradle" version "1.4.0"
+	id 'java-library'
+	id 'scala'
+	id 'maven-publish'
+	id 'fabric-loom' version '1.5-SNAPSHOT'
 }
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
-
 archivesBaseName = project.archives_base_name
 group = project.maven_group
 
-version = mod_version
-def ENV = System.getenv()
-version = version + "+scala." + project.scala_version + (ENV.GITHUB_ACTIONS ? "" : ".local")
+version = mod_version + "+scala." + project.scala_version + (System.getenv('GITHUB_ACTIONS') ? "" : ".local")
 
 dependencies {
-    minecraft "com.mojang:minecraft:${project.minecraft_version}"
-    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	minecraft "com.mojang:minecraft:${project.minecraft_version}"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-	include(api("org.scala-lang:scala3-library_3:${project.scala_version}"))
-	// Depended on by scala3
-	include(api("org.scala-lang:scala-library:2.13.6"))
+	// Scala Dependencies
+	implementation "org.scala-lang:scala3-library_3:${project.scala_version}"
```

Comment:
Does it seems right? I don't think this is the correct nested-jar.

After I testing about it, I can't find any bundled jar in `fabric-language-scala`.

@modmuss50 

Maybe correcting it into 

```
modApi(include("org.scala-lang:scala3-library_3:${project.scala_version}"))
```
like [`fabric-language-kotlin`](https://github.com/FabricMC/fabric-language-kotlin/blob/424244fd4ac34bc87cd0402913b344da55754624/build.gradle#L62C1-L69C2) will be ok?

![image](https://github.com/FabricMC/fabric-language-scala/assets/20512501/248582f5-bf8f-4393-b184-e050098c4864)
![image](https://github.com/FabricMC/fabric-language-scala/assets/20512501/f6e7428f-379e-4504-8434-970b594e7ba7)